www/class/centreon-clapi/centreonHost.class.php

```diff
@@ -1435,8 +1435,7 @@ public function export($filterName = null)
             "AND"
         );
         foreach ($elements as $element) {
-            $exportContactName = isset($element['contact_name']) ? $element['contact_name'] : null;
-            CentreonContact::getInstance()->export('CONTACT', $exportContactName);
+            CentreonContact::getInstance()->export($element['contact_alias']);
```

Comment:
isn't first argument missing ? ('CONTACT')

Reply:
There is only one argument, which should be the value of the unique field of the objet : 
https://github.com/centreon/centreon/blob/master/www/class/centreon-clapi/centreonContact.class.php#L483